### ISSUE TITLE
Adding an image or Screenshot in the "Notes" section

### ISSUE THREAD

**Issue by DukeBurkoff** [Fate-Grand-Automata/FGA#810]
Before opening an issue:
- [x] No one has asked for this feature before in other issues (even the closed ones)

> If you open unnecessary/dumb issues, you'll be rewarded with appropriate tags.

I usually use the "Notes section" in Fga a lot  but since I dont have any of the meta supports i usually have to get creative in team building but with such Events with multiples sections or constantly changing farming missions writing every detail for every team everytime is usually a hassle specially since i usually just make a team and autobattle while im working, and i think just adding an option to add a image there (a screenshot of the team) would solve it and would be really helpful for me and others in saving time in that aspect.

I also wanted to talk about a minor issue but is so small i didnt want to just open another :issue: for such a minor problem
-About FGA not automatically starting a mission if this one doesn't have a fight or the fight is after the story and there is a  confirmation box (shown in screenshot), i think this one would be especially useful in situations where you are rushing through the story or events specially when there are Raids.

I dont think these problems require any priority since they are so minor but even so im really grateful for your attention and your amazing work in FGA.
![Screenshot](https://user-images.githubusercontent.com/81212715/128597924-2aaa2921-984f-4229-8f5b-942c0e656117.jpg)

**Comment by MathewSachin** [comment]
Adding images to Battle configs is not practical due to how the configs are stored on disk. They're stored in SharedPreferences which is a key value pair store which can store primitive data like numbers and text but nothing complex like complete files, audio, images or video.
We could encode the image as a base64 string but the string size would be too large in most cases to justify.
Also, configs are imported and exported as JSON files, so the approach of keeping attached images on disk and linking with the config won't work either.

**TL;DR:** I don't think it's possible with how the app works.

Regarding the second request:
- We don't focus on things that only matter for doing story, unless it is really simple to do.
- That dialog would be different per server. Too much effort to maintain with little returns.
